src/components/reportingCards/ReportingCard.js

```diff
@@ -228,9 +236,15 @@ function GrowerReportingCard(props) {
                   <Icon className={classes.iconTotal} style={{ color }} />
                 </Box>
                 <Box className={classes.box3}>
-                  <Typography className={classes.total}>
-                    {new Intl.NumberFormat().format(data.num1)}
-                  </Typography>
+                  {typeof data.num1 === 'string' ? (
```

Comment:
As far as I can tell, all these stats come through from the API as strings and are formatted as locale-specific numbers using `Intl.NumberFormat().format()`.

Reply:
Same goes for the other places this has changed.

Reply:
I tried to simplify this now so that we can have the international number formats, and the percentage if it is included.